content/doc/developer/publishing/documentation.adoc

```diff
@@ -0,0 +1,169 @@
+---
+title: Plugin Documentation
+layout: developersection
+---
+
+Documentation is an important part of any Jenkins plugin.
+It includes user documentation (plugin pages, changelogs, user guidelines, etc.)
+and the contributor documentation (how to contribute, developer guidelines, etc.).
```

Comment:
This part of jenkins.io generally uses line-per-sentence formatting, I would appreciate if you could keep this consistent.